getting-started/glossary.md

```diff
@@ -1,51 +1,93 @@
-# GLOSSARY   
+# GLOSSARY
 
-## API
-Application Programming Interface. A software intermediary that allows two applications to talk to each other.
+This page provides a brief explanation of the key terms used in Cloud File Transfer  
 
+## API
+Application Programming Interface. Refers to a set of functions and procedures that allows two applications to interact with each other programmatically in order to access the features or data of an operating system, application or other service. 
 
 ## API Endpoint
-The place that APIs send requests to and where the resource resides.
+Refers to a URL of a server or service, that is, the location from which APIs can access the API resources that they need to carry out the function.
 
 ## API Gateway
-An API management tool that sits between a client and a collection of backend services.
-
-## API Gateway Id
-
+An API management tool that sits between a client and a collection of backend services. It accepts all application programming interface (API) calls, aggregates the various services required to fulfill them, and returns the appropriate result.
```

Comment:
"It accepts all API calls, aggregates the various services required to fulfil them, and returns the appropriate result."

--- dont need to expand API and edited"fulfil"